bindings/pyroot_experimental/cppyy/CPyCppyy/src/CPPInstance.h

```diff
@@ -98,7 +98,12 @@ class CPPInstance {
 
 
 //- object proxy type and type verification ----------------------------------
```

Comment:
To keep things in sync, the changes in this file and in `CPPOverload.h` should be submitted as a PR to the CPyCppyy repo:
https://bitbucket.org/wlav/cpycppyy
and we can get the changes in the next CPyCppyy release.

Reply:
Well, it was already part of the changes I submitted to Wim, but OK, I'll try a to make a PR in his repo